enso/experiment/finetuning.py

```diff
@@ -37,6 +53,127 @@ def predict(self, X, **kwargs):
+
+@Registry.register_experiment(ModeKeys.CLASSIFY, requirements=[("Featurizer", "PlainTextFeaturizer")])
+class Finetune2Summative(Finetune):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.model.config.trainable_layers = [True] * 2
+        self.model.config.trainable_old_embeddings = False
+        self.model.config.trainable_new_embeddings = False
+        self.model.config.init_embeddings_from_file = "embeddings.npy"
+        self.model.config.reduce_states="SUM"
+
+
+@Registry.register_experiment(ModeKeys.CLASSIFY, requirements=[("Featurizer", "PlainTextFeaturizer")])
+class Finetune2Mean(Finetune):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.model.config.trainable_layers = [True] * 2
+        self.model.config.trainable_old_embeddings = False
+        self.model.config.trainable_new_embeddings = False
+        self.model.config.init_embeddings_from_file = "embeddings.npy"
+        self.model.config.reduce_states="Mean"
+
+
+@Registry.register_experiment(ModeKeys.CLASSIFY, requirements=[("Featurizer", "PlainTextFeaturizer")])
+class Finetune4(Finetune):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.model.config.trainable_layers = [True] * 4
+        self.model.config.trainable_old_embeddings = False
+        self.model.config.trainable_new_embeddings = False
+        self.model.config.init_embeddings_from_file = "embeddings.npy"
+
+
+@Registry.register_experiment(ModeKeys.CLASSIFY, requirements=[("Featurizer", "PlainTextFeaturizer")])
+class Finetune6(Finetune):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.model.config.trainable_layers = [True] * 6
+        self.model.config.trainable_old_embeddings = False
+        self.model.config.trainable_new_embeddings = False
+        self.model.config.init_embeddings_from_file = "embeddings.npy"
+
+
+@Registry.register_experiment(ModeKeys.CLASSIFY, requirements=[("Featurizer", "PlainTextFeaturizer")])
+class Finetune8(Finetune):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.model.config.trainable_layers = [True] * 8
+        self.model.config.trainable_old_embeddings = False
+        self.model.config.trainable_new_embeddings = False
+        self.model.config.init_embeddings_from_file = "embeddings.npy"
+
+
+@Registry.register_experiment(ModeKeys.CLASSIFY, requirements=[("Featurizer", "PlainTextFeaturizer")])
+class Finetune10(Finetune):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.model.config.trainable_layers = [True] * 10
+        self.model.config.trainable_old_embeddings = False
+        self.model.config.trainable_new_embeddings = False
+        self.model.config.init_embeddings_from_file = "embeddings.npy"
+
```

Comment:
Do we actually want all the of these experiments in the registry, or were they mainly for a one-time thing?